src/commands/BootstrapCommand.js

```diff
@@ -205,7 +205,7 @@ export default class BootstrapCommand extends Command {
           // get Package of dependency
           const dependencyPackage = this.packageGraph.get(dependency).package;
           // get path to dependency and its scope
-          const { location: dependencyLocation, scope: dependencyScope } = dependencyPackage;
+          const { location: dependencyLocation } = dependencyPackage;
```

Comment:
found it while trying to figure how to destructure without conflicting with an already existing variable with the same name. cool stuff!